NOTICE.md

```diff
@@ -0,0 +1,19 @@
+## NOTICE
+
+Copyright (c) 2023-2024, Owners of https://github.com/autogen-ai
+
+This project is a fork of https://github.com/microsoft/autogen.
+
+The [original project](https://github.com/microsoft/autogen) is licensed under the MIT License as detailed in [LICENSE_original_MIT](./license_original/LICENSE_original_MIT). The fork was created from version v0.2.35 of the original project.
+
+
+This project, i.e., https://github.com/autogen-ai/autogen, is licensed under the Apache License, Version 2.0 as detailed in [LICENSE](./LICENSE)
+
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+
+Ongoing MIT-licensed contributions:
+This project regularly incorporates code merged from the [original repository](https://github.com/microsoft/autogen) after the initial fork. This merged code remains under the original MIT license. For specific details on merged commits, please refer to the project's commit history.
```

Comment:
This looks complicated. Can we avoid the burden of checking commit history on a user?

Reply:
The users don't need to check the commit history. The ones who want to know details need to check.